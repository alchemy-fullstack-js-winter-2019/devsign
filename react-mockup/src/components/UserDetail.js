import React from 'react';
import UserInfo from './UserInfo';
import HumsByUser from './HumsByUser';
import Header from './Header';
import Footer from './Footer';

function UserDetail() {
  return (
    <>
    <Header />
    <main>
      <UserInfo />
      <HumsByUser />
    </main>
    <Footer />
    </>
  );
}

export default UserDetail;
