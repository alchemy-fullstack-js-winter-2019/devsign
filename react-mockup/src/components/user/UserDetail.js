import React from 'react';
import UserInfo from './UserInfo';
import HumsByUser from './HumsByUser';
import Header from '../app/Header';
import Footer from '../app/Footer';
import HumForm from '../hums/HumForm';

function UserDetail() {
  return (
    <>
    <Header />
    <main>
      <UserInfo />
      <HumForm />
      <HumsByUser />
    </main>
    <Footer />
    </>
  );
}

export default UserDetail;
