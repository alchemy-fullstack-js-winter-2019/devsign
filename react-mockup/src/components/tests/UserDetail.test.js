import React from 'react';
import renderer from 'react-test-renderer';
import UserDetail from '../user/UserDetail';
import { MemoryRouter } from 'react-router';

describe('UserDetail snapshot', () => {
  it('can match a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <UserDetail />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
