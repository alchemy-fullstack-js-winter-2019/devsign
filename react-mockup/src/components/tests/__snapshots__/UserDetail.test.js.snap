// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDetail snapshot can match a snapshot 1`] = `
Array [
  <header
    className="sc-htpNat gwiVQj"
  >
    <div
      className="sc-ifAKCX dakLHt"
    >
      <h1>
        hummingbird
      </h1>
      <nav
        className="sc-bxivhb gJQaCU"
      >
        <ul>
          <li>
            <a
              href="/"
              onClick={[Function]}
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/hum-feed"
              onClick={[Function]}
            >
              HumFeed
            </a>
          </li>
          <li>
            <a
              href="/user"
              onClick={[Function]}
            >
              User
            </a>
          </li>
        </ul>
      </nav>
    </div>
    <img
      alt="Hummingbird logo"
      src="react-mockup/src/assets/logo.png"
    />
  </header>,
  <main>
    <section
      className="sc-bdVaJa jfWfOm"
    >
      <img
        alt="user icon"
        src="react-mockup/src/assets/user-image.png"
        width="100px"
      />
      <h3>
        @username
      </h3>
      <p>
        Location, Country
      </p>
      <p>
        website: 
        <a
          alt="user website"
          href="/"
        >
          website
        </a>
      </p>
      <p>
        Bio. A little bit about user 247.
      </p>
    </section>
    <div
      className="sc-bZQynM jVJWHm"
    >
      <form>
        <label
          htmlFor="hum"
        >
          <p>
            post a new hum
          </p>
          <input
            name="hum"
            type="text"
          />
        </label>
        <button>
          add new hum
        </button>
      </form>
    </div>
    <section
      className="sc-bwzfXH cTYrYG"
    >
      <h3>
        hums by you!
      </h3>
      <div>
        <p>
          Hum hum hum. bother. bother. bother.
        </p>
      </div>
      <div>
        <p>
          Hum hum hum. bother. bother. bother.
        </p>
      </div>
      <div>
        <p>
          Hum hum hum. bother. bother. bother.
        </p>
      </div>
    </section>
  </main>,
  <footer
    className="sc-EHOje eWyvtp"
  >
    <p>
      Â© Hummingbird, 2019
    </p>
  </footer>,
]
`;
